### ISSUE TITLE
box loss become nan and i found where loss become nan

### ISSUE THREAD

**Issue by Penuoz** [jwyang/faster-rcnn.pytorch#381]
I used my own dataset training and had the same problem as others: loss became nan.

So I debugged and found that in the _smooth_l1_loss function, the loss_box was originally a tensor of the [2, 60, 64, 38] dimension, and the value of each bit was 1e-2*0.

     For i in sorted(dim, reverse=True):
       Loss_box = loss_box.sum(i)

**After this operation, the value of loss_box becomes nan, causing the entire loss to become nan.** I found the problem, but I don't know how to solve it? In my imagination, the value of loss_box.sum should be 0, why is it nan?

**Comment by waleedfarrukhgini** [comment]
Hi @Penuoz,
Did you find a way around it?
